### PR TITLE
Add Katello 4.14 API docs

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@ianballou how does this look?

File: index.html [hunk 1]
```diff
@@ -30,6 +30,7 @@ <h2>Foreman core API</h2>
 	<h2>Katello API</h2>
 	<ul>
 		<li><a href="./katello/latest/apidoc/v2.html">latest</a></li>
+                <li><a href="./katello/4.13/apidoc/v2.html">4.14</a></li>
```
Comment (ianballou) [inline index.html]:
```suggestion
                <li><a href="./katello/4.14/apidoc/v2.html">4.14</a></li>
```

**Comment by chris1984** [comment]
@ianballou updated if you want to take another look

**Review by ianballou — APPROVED** [review]
Looking good :+1:

**Pull request merged**